dotcom-rendering/src/lib/getFrontsAdPositions.ts

```diff
@@ -296,13 +297,17 @@ const canInsertDesktopAd = (
 const getFrontsBannerAdPositions = (
 	collections: DCRCollectionType[],
 	pageId: string,
-): number[] =>
-	collections.reduce<{ heightSinceAd: number; adPositions: number[] }>(
+): number[] => {
+	const maxAdsAllowed = hasSecondaryLevelContainers(collections)
+		? MAX_FRONTS_BANNER_ADS_BETA
+		: MAX_FRONTS_BANNER_ADS;
+
+	return collections.reduce<{ heightSinceAd: number; adPositions: number[] }>(
```

Comment:
Given this uses heightSinceAd to calculate, would it makes sense to increase the height of a scrollable/small in this PR as well?

scrollable/small now takes up the same space as a scrollable/medium (0.5 -> 1)